skimage/exposure/exposure.py

```diff
@@ -144,15 +144,20 @@ def histogram(image, nbins=256, source_range='image', normalize=False):
     return hist, bin_centers
 
 
-def cumulative_distribution(image, nbins=256):
+def cumulative_distribution(image, nbins=256, *, source_range='image'):
     """Return cumulative distribution function (cdf) for the given image.
 
     Parameters
     ----------
     image : array
         Image array.
     nbins : int, optional
-        Number of bins for image histogram.
+        Number of bins used to calculate histogram. This value is ignored for
+        integer arrays, for which each integer is its own bin.
+    source_range : string, optional
+        'image' (default) determines the range from the input image.
```

Comment:
```suggestion
        'image' (default) determines the range from minimum and maximum value of the input image.
```
is this true?

Reply:
Yes, it's true, but not PEP8 compliant. =P

Reply:
I'm advocating to have this option set to 'source' by default. Although 'image' can be useful, it is not the most natural option to me. Especially, once you want to compare histograms, different ranges do not help.

Reply:
@hmaarrfk Added and modified elsewhere also.